Document/0x05b-Basic-Security_Testing.md

```diff
@@ -10,16 +10,16 @@ At the very least, you'll need [Android Studio](https://developer.android.com/st
 
 #### Setting up the Android SDK
 
-Local Android SDK installations are managed via Android Studio. Create an empty project in Android Studio and select "Tools->Android->SDK Manager" to open the SDK Manager GUI. The "SDK Platforms" tab is where you install SDKs for multiple API levels. Recent API levels:
+Local Android SDK installations are managed via Android Studio. Create an empty project in Android Studio and select "Tools->Android->SDK Manager" to open the SDK Manager GUI. The "SDK Platforms" tab is where you install SDKs for multiple API levels. Recent API levels are:
 
-- API 23: Android 6.0
-- API 24: Android 7.0
-- API 25: Android 7.1
-- API 26: Android 8.0
-- API 27: Android 8.1
-- API 28: Android 9.0
+- Android 9.0 (API level 28)
+- Android 8.1 (API level 27)
+- Android 8.0 (API level 26)
+- Android 7.1 (API level 25)
 
-![SDK Manager](Images/Chapters/0x05c/sdk_manager.jpg)
+An overview of all Android codenames, it's version number and API Levels can be found in the [Android Developer Documentation](https://source.android.com/setup/start/build-numbers "Codenames, Tags, and Build Numbers").
```

Comment:
it's -> their (and it should have been "its" :) )

Reply:
haha, true :-) done